mc_mitm/source/controllers/atgames_controller.hpp

```diff
@@ -79,6 +82,7 @@ namespace ams::controller {
 
         private:
             void MapInputReport0x01(const AtGamesReportData *src);
+            bool m_arcadepanel = false;
```

Comment:
Don't initialise member variables here, do it in the class constructor. Preferably as part of an initialiser list to be consistent with the other classes.

Also separate the method and variable definitions with a blank line.